packages/firestore/src/api/credentials.ts

```diff
@@ -562,6 +569,9 @@ export class FirebaseAppCheckTokenProvider
   }
 
   getToken(): Promise<Token | null> {
+    if (this.serverAppAppCheckToken !== null) {
```

Comment:
probably just want truthy here to handle `undefined`

Reply:
it should be find with just null, as that's what it's defaulted to but it seems a foot gun

Reply:
Done!